.github/workflows/build.yml

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
+#
+# SPDX-License-Identifier: BSD-2-Clause
+
+# Build and deploy the site
+
+name: Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  schedule:
+    - cron: '7 3 * * *'
```

Comment:
You sure about this?  If that's standard crontab notation you're asking for a run at 7 minutes past 3 every morning.

Reply:
Yes, that was the plan. This idea is to run once nightly even if there are no changes to this repo, so that it can pick up changes to pulled-in docs that might have happened in other repos.